src/Client/AddClient.php

```diff
@@ -8,15 +8,20 @@
 use RialtoWebService\StructType\AddRialtoOrders;
 use RialtoWebService\StructType\RialtoWebOrderList;
 use WsdlToPhp\PackageBase\AbstractSoapClientBase;
+use Psr\Log\LoggerInterface;
 
 class AddClient
 {
     /** @var ClientConfiguration */
     private $configuration;
 
-    public function __construct(ClientConfiguration $configuration)
+    /** @var LoggerInterface */
+    private $loggerInterface;
```

Comment:
The property should be called `$logger`, not `$loggerInterface`. The same goes for the constructor parameter.

This also applies to `GetClient`.